QUEST_LV_0100.tsv

```diff
@@ -7906,146 +7906,146 @@ QUEST_LV_0100_20150804_007901	$Historian Rexipher is thinking deeply about somet
-QUEST_LV_0100_20150908_008025	Retrieving the magical crystal
-QUEST_LV_0100_20150908_008026	Defeat Meroggs and Chafferors that followed Broknews!
+QUEST_LV_0100_20150908_008018	$Defeat Chapparitions that are disturbing when you are checking the tombstone{nl}Check what was written on the last tombstone of Lydia Schaffen
+QUEST_LV_0100_20150908_008019	$You've read a tombstone of Agailla Flurry{nl}Read the other tombstones to obtain the artifacts of Flurry
+QUEST_LV_0100_20150908_008020	$The remaining parts are unreadable
+QUEST_LV_0100_20150908_008021	$You've read a tombstone of Agailla Flurry{nl}Read the other tombstones to obtain the artifacts of Flurry
+QUEST_LV_0100_20150908_008022	$You've read all of Agailla Flurry's epitaphs{nl}Your status has increased by 1!
+QUEST_LV_0100_20150908_008023	$Defeat Zolem and collect Zolem Magic Stones
+QUEST_LV_0100_20150908_008024	$Conveying Cahill's Feelings
+QUEST_LV_0100_20150908_008025	$Retrieving the Magic Crystal
+QUEST_LV_0100_20150908_008026	$Defeat the Merogs and Chafperors that followed Broknews!
 QUEST_LV_0100_20150908_008027	{memo X}
 QUEST_LV_0100_20150908_008028	{memo X}
-QUEST_LV_0100_20150908_008029	You have activated the altar of purification!
-QUEST_LV_0100_20150908_008030	You have collected all Wood Saps.{nl}Go to Ershike Altar to combine the ingredients!
-QUEST_LV_0100_20150908_008031	Combining the ingredients
-QUEST_LV_0100_20150908_008032	The dye has been completed!{nl}Go to the altar to obtain the dye
-QUEST_LV_0100_20150908_008033	Return to the Andale Village Priest
-QUEST_LV_0100_20150908_008034	Find the Andale Village Priest in Cobalt Forest!
-QUEST_LV_0100_20150908_008035	Moldyhorn has appeared!
-QUEST_LV_0100_20150908_008036	Combining the ingredients
-QUEST_LV_0100_20150908_008037	Defeat the monsters that reacted to the ominous energy!
-QUEST_LV_0100_20150908_008038	Blue Woodspirit has appeared!
-QUEST_LV_0100_20150908_008039	Obtain Languid Herb from Narvas Curved Path
-QUEST_LV_0100_20150908_008040	Look for strongly scented soul flower at Dvyni Wetland
-QUEST_LV_0100_20150908_008041	You have obtained strongly scented soul flower!{nl}Return to the Andale Village Priest
-QUEST_LV_0100_20150908_008042	Find the headman of the village
-QUEST_LV_0100_20150908_008043	Bring the bomb from one of the barrels located at the upper side of the village
-QUEST_LV_0100_20150908_008044	Look for tools that can be used to pull out the object in the well!
+QUEST_LV_0100_20150908_008029	$You've activated the altar of purification!
+QUEST_LV_0100_20150908_008030	$You've collected all the resin of the trees.{nl}Go to Ershike Altar and combine the ingredients!
+QUEST_LV_0100_20150908_008031	$Combining the ingredients
+QUEST_LV_0100_20150908_008032	$The dye has been completed!{nl}Go to the altar and obtain the dye
+QUEST_LV_0100_20150908_008033	$Return to the Andale Village Priest
+QUEST_LV_0100_20150908_008034	$Meet the Andale Village priest at Cobalt Forest!
```

Comment:
Previous version works just fine, because finding him is what you do first. Priest should be capitalized.